package.json

```diff
@@ -64,43 +64,51 @@
   },
   "homepage": "https://github.com/yarinvak/polaris#readme",
   "dependencies": {
+    "@enigmatis/mongo-driver": "1.8.0",
```

Comment:
is it possible to put only in dev dependencies or something like that

Reply:
If its for the integration test, shouldn't it be in a different package.json? (separated server inside test package like in mgf)

Reply:
its should be only in dev dependencies